.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
I don't mind having this in the repo, but just a tip: You can add a global ignore file for git to ignore .DS_Store files for all your repos. I have it set up like this

in ~/.gitconfig
```
[core]
        excludesfile = ~/.gitignore
```
and in ~/.gitignore
```
.*.swp
env/
.ruby-version
.ruby-gemset
*.pgdump
```

Pretty handy to not have to set up the same ignore files for every repo.

Reply:
OH this is good. I'll remove this file from the repo.